### ISSUE TITLE
Nuclear too flexible?

### ISSUE THREAD

**Issue by GordStephen** [GridMod/RTS-GMLC#140]
The nuclear unit ramp rate is given as 20 MW/min, about 5x higher than the comparably-sized gas CCs and coal units. Is that intentional?

**Comment by claytonpbarrows** [comment]
not really, but the pmin is 396MW and pmax is 400, so while it may have really relaxed ramp rates, it can't go very far.

**Comment by claytonpbarrows** [comment]
However, I would be happy to tighten them to something more realistic.
